### DISCUSSION TITLE
Multiselect when Select all/Unselect all should let disabled fields without modification

### DISCUSSION THREAD

**Discussion by jpeter06** [primefaces/community#1252] [PrimeNG]
Hello All,
When you execute in **Multiselect** _select/unselect all_, the **disabled** options **should not change**. (If not, this allow to change not editable options)

thanks for all

**Comment by cetincakiroglu** [comment]
Hi,

Couldn't replicate the behavior, disabled options are still the same please see the example: https://stackblitz.com/edit/3csuin?file=src%2Fapp%2Fdemo%2Fmulti-select-basic-demo.ts

If you're having the issue, please share a stackblitz example so we can identify the issue.

**Comment by cetincakiroglu** [comment]
Closing the discussion due to inactivity.
